sdk/src/utils/fees/estimateOraclePriceCount.ts

```diff
@@ -0,0 +1,24 @@
+// @see https://github.com/gmx-io/gmx-synthetics/blob/6ed9be061d8fcc0dc7bc5d34dee3bf091408a1bf/contracts/gas/GasUtils.sol#L218-L234
+export function estimateDepositOraclePriceCount(swapsCount: number): bigint {
+  return 3n + BigInt(swapsCount);
+}
+
+export function estimateWithdrawalOraclePriceCount(swapsCount: number): bigint {
+  return 3n + BigInt(swapsCount);
+}
+
+export function estimateOrderOraclePriceCount(swapsCount: number): bigint {
```

Comment:
only this function is in use, do you think it makes sense to keep the rest functions?

Reply:
Eventually the rest will be here, so I think it's ok to move fees utils files entirely